config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -4,7 +4,7 @@ presubmits:
     cluster: eks-prow-build-cluster
     branches:
     - ^main
-    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$|^(keps|cmd/experimental/kjobctl|site|charts)/"
```

Comment:
Let's change Kueue in a follow up. Here we focus on kjob

Reply:
Done